README.md

```diff
@@ -148,7 +148,12 @@ Then configure your bridge with the audience of the target service:
 use BridgeEx.Graphql,
   endpoint: "...",
   decode_keys: :strings,
-  auth0: [enabled: true, audience: "target_audience"]
+  auth0: [
+    enabled: true,
+    audience: "target_audience",
+    # You can optionally specify a client (in case you defined multiple ones)
```

Comment:
What about saying that the default client will be used otherwise?

Reply:
Wdyt?

Reply:
:+1: